testsuite/libra-swarm/src/swarm.rs

```diff
@@ -74,14 +73,10 @@ impl LibraNode {
             // Only set our RUST_LOG if its not present in environment
             node_command.env("RUST_LOG", "debug");
         }
-        if disable_logging {
-            node_command.arg("-d");
-        } else {
-            node_command.env("STRUCT_LOG_FILE", struct_log_path);
-        }
         node_command
             .stdout(log_file.try_clone()?)
-            .stderr(log_file.try_clone()?);
+            .stderr(log_file.try_clone()?)
+            .env("STRUCT_LOG_FILE", struct_log_path);
```

Comment:
We don't even respect this env variable anymore...I think?

Reply:
more to delete!